README.md

```diff
@@ -71,7 +75,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+This code would most make sense as a model method. This is to practice separation of concerns and follow the rails pattern "Fat Model, Skinny Controller". Since changing a color doesn't require directly accessing the database, it's not needed in the controller. As a model method, it can be accessed in the view, passing it the data the controller provided the view.
 ```
 
```

Comment:
In this example, it would make the most sense to use a Helper method, because these are intended to "help" render HTML based on conditions. Controller methods concern how data moves between routes. Model methods concern how data goes to and comes from the database.